apps/meteor/app/apps/server/communication/rest.js

```diff
@@ -885,5 +885,34 @@ export class AppsRestApi {
 				},
 			},
 		);
+
+		this.api.addRoute(
+			'app-request',
+			{ authRequired: true },
+			{
+				async get() {
+					const baseUrl = orchestrator.getMarketplaceUrl();
+					const { appId, q, sort, limit, offset } = this.queryParams;
```

Comment:
Let's apply some defaults here, since it is an api endpoint and we are immediately using the values below.

Reply:
```suggestion
					const { appId, q: '', sort: '', limit: 25, offset: 0 } = this.queryParams;
```